throws
    用在方法声明后面，跟的是异常类名
    可以跟多个异常类名，用逗号隔开
    表示抛出异常，由该方法的调用者来处理
    throws表示出现异常的一种可能性，并不一定会发生这些异常

throw
    用在方法体内，跟的是异常对象名
    只能抛出一个异常对象名
    表示抛出异常，由方法体内的语句处理
    throw则是抛出了异常，执行throw则一定抛出了某种异常

原则:
    如果该功能内部可以将问题处理,用try,如果处理不了,交由调用者处理,这是用throws
区别:
    后续程序需要继续运行就try
    后续程序不需要继续运行就throws